llvm_jni/modules/jni.llvm/LLVMUtils.bal

```diff
@@ -0,0 +1,66 @@
+import ballerina/jballerina.java;
+
+distinct class PointerPointer {
```

Comment:
What's a PointerPointer? This needs a comment (and probably a better name).

Reply:
[`PointerPointer`](https://www.javadoc.io/static/org.bytedeco/javacpp/1.3.3/org/bytedeco/javacpp/PointerPointer.html) is how they have handled void pointers for arrays. For example 
``` LLVMValueRef LLVMBuildCall(LLVMBuilderRef B, LLVMValueRef Fn, LLVMValueRef *Args, unsigned NumArgs,                        const char *Name) ```
 is the C-API translate to 
```public static native LLVMValueRef LLVMBuildCall(LLVMBuilderRef arg0, LLVMValueRef Fn,                     @Cast("LLVMValueRef*") PointerPointer Args, @Cast("unsigned") int NumArgs, String Name)```
 in the Java API